mxcube3/__init__.py

```diff
@@ -83,10 +83,11 @@ def complete_initialization(app):
         app.collect = app.beamline.getObjectByRole("collect")
         app.diffractometer = app.beamline.getObjectByRole("diffractometer")
 	try:
-	    app.diffractometer.centring_motors_list
+	    if getattr(app.diffractometer, 'centring_motors_list') == None:
```

Comment:
I added this code for backward compatibility. We are not on 2.2 here, and there is no `centring_motors_list` in our diffractometer objects. I need this code to be able to continue with my tests and development on the ESRF beamlines.

In its current state I cannot merge the PR because it is missing the third argument to `getattr` : at the moment it would raise the `AttributeError` exception on a diffractometer object without `centring_motors_list`.